### ISSUE TITLE
[Scene Mesh] Traces pass through Meshes unless you wiggle the Collision Type

### ISSUE THREAD

**Issue by Retroeer** [Facepunch/sbox-issues#7257]
### Describe the bug

If you have a basic Trace pointed at a piece of Scene Mesh, it will pass straight through it until you switch the Collision Type

### To Reproduce

1. Make a Scene Mesh box or something
2. Trace for it
3. Should return nothing until you jiggle the collision type

### Expected behavior

Traces should always hit the mesh

### Media/Files

https://github.com/user-attachments/assets/0ecde8f3-4811-4254-af97-734c047b3596

**Comment by aylaylay** [comment]
Make a repro project because the collision is there and I can trace it no problem

**Comment by Retroeer** [comment]
yeah sorry, took a break from it, this seems to only occur when the trace is only searching for a tag with .WithTag

If you remove the .WithTag it actually hits it just fine, I'm assuming the reason wiggling it works is because the tag is getting properly applied?

**Comment by aylaylay** [comment]
upload a repro

**Comment by Retroeer** [comment]
turns out its due to me adding a tag on start, if you add a simple component to the mesh and add a tag to it on start, then trace for that tag, then it doesn't detect it

**Comment by aylaylay** [comment]
upload a repro

**Comment by Retroeer** [comment]
[colliderrepro.zip](https://github.com/user-attachments/files/18285025/colliderrepro.zip)

**Comment by aylaylay** [comment]
Thanks - fixed
